leonicaisse/adeo-test
# feat: improve error handling

# Description
This PR includes improvements for error handling in the app, and a small refactoring in order to make it easier to test the application.

# How to test
Every single point of the README should work as intended :
Run `node app.js --count`
Run `node app.js --filter=ry`
Run `npm test`
